### PR TITLE
Fix broken links, add `html_meta` values

### PR DESCRIPTION
**Pull request by stevepiercy**
See https://github.com/plone/documentation/pull/1195

See also https://github.com/plone/volto/issues/3128

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -42,9 +46,15 @@
 
 ### Internal
 
+- Various build improvements and minor refactoring captured during the effort to unify
```
Comment (sneridagh) [inline CHANGELOG.md:49]: Where does this come from? Merge issue?

Comment (stevepiercy) [reply CHANGELOG.md:49]: Yes. GitHub complained about a bad merge commit, and I attempted to resolve it. I'm not sure exactly where it belongs, so I made a best guess.

Comment (stevepiercy) [reply CHANGELOG.md:49]: Heh, and the changelog.md is out of date again. I think I'll let the release manager handle that. :wink:

Comment (sneridagh) [reply CHANGELOG.md:49]: Ok no worries! I'll take a look!

Comment (sneridagh) [reply CHANGELOG.md]: Done

Comment (stevepiercy) [reply CHANGELOG.md]: Thank you!

**Pull request merged**